### ISSUE TITLE
Supporting interactive epidemiology data apps w/ R and Shiny

### ISSUE THREAD

**Issue by daguar** [codeforamerica/project-ideas#29]
@jmadans passed along a note from an epidemiologist in the public health world who was using...
- [R — open source stats software, similar to SAS/SPSS)](http://www.r-project.org/), and
- [Shiny](http://shiny.rstudio.com/) — an R plug-in that generates interactive web apps from R code

to create interactive web interfaces to epi data. He was looking for a Linux server to deploy to, but this sparked a broader idea to me — providing the support infrastructure (documentation + active outreach/dissemination) for epidemiologists to use R + Shiny to deploy their own web interactives with the data they are holding onto.

To me, this is really about capacity-building inside public health departments by connecting folks to a world they simply often don't know about yet.

Some elements of this could be:
- A splash page with the a nice path for getting started with R (using the external resources that exist already)
- Documentation on deploying to Heroku (or the new [ShinyApps.io](https://www.shinyapps.io/))
- A "gallery" page of cool Shiny apps developed by epidemiologists

Basically my hunch here is that this is how we bridge that domain divide — so many of the people in public health know SAS, and R isn't a huge leap. And once you know R, Shiny is even less of a leap. And if we make it dead-easy for them to know how they can publish their R work to the world, everyone wins.

FIN.

cc @migurski

**Comment by daguar** [comment]
cc @semerj — Want to start working on a tutorial for deploying to both ShinyApps.io and heroku?

**Comment by semerj** [comment]
yeah, let's do this

**Comment by haroldgil** [comment]
@daguar  Hi, I was wondering if this idea for a project was pursued.

**Comment by RFNajera** [comment]
So... Did you guys do this?
